src/maturin_import_hook/settings.py

```diff
@@ -99,7 +99,7 @@ def to_args(self) -> list[str]:
 class MaturinBuildSettings(MaturinSettings):
     """settings for `maturin build`."""
 
-    skip_auditwheel: bool = False
+    auditwheel: Optional[str] = None
```

Comment:
technically this means the import hook doesn't have 100% compatibility with older maturin versions but I don't think it's worth (A) maintaining backwards compatibility or (B) bumping the minimum supported version

Reply:
I think it's fine.